tests/src/main/java/com/hazelcast/stabilizer/tests/map/SSLEnterpriseTest.java

```diff
@@ -0,0 +1,113 @@
+package com.hazelcast.stabilizer.tests.map;
+
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.stabilizer.test.TestContext;
+import com.hazelcast.stabilizer.test.TestRunner;
+import com.hazelcast.stabilizer.test.annotations.Run;
+import com.hazelcast.stabilizer.test.annotations.Setup;
+import com.hazelcast.stabilizer.test.annotations.Teardown;
+import com.hazelcast.stabilizer.test.annotations.Verify;
+import com.hazelcast.stabilizer.test.utils.ThreadSpawner;
+import com.hazelcast.stabilizer.tests.helpers.KeyLocality;
+import com.hazelcast.stabilizer.tests.helpers.KeyUtils;
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.Arrays;
+import java.util.Random;
+
+import static com.hazelcast.stabilizer.test.utils.TestUtils.assertEqualsByteArray;
+
+/**
+ * This test provides functionality of stressing for SSL feature of Hazelcast Enterprise
+ */
+public class SSLEnterpriseTest {
+
+    private final static ILogger LOGGER = Logger.getLogger(SSLEnterpriseTest.class);
```

Comment:
use same order final static vs static final.

Make sure you treat your code with care. If you make messy code, the you don't respect your own work.